src/data/sql-quiz.ts

```diff
@@ -1033,6 +1033,19 @@ const sqlQuiz = [
     Explanation:
       "To select some particular date between one date and another, you can use the BETWEEN and AND keywords while specifying the dates. You can also use comparison operators such as greater than (>), less than (<), or equal(=).",
     Link: "https://www.freecodecamp.org/news/sql-date-function-query-timestamp-example-format/"
+  },
+  {
+    Question:
+      "Which of the following SQL queries correctly shows the number of unique subjects taken by a student?",
+    Answer: "COUNT(DISTINCT SUBJECT_ID)",
+    Distractor1:
+      "COUNT(UNIQUE SUBJECT_ID)",
+    Distractor2:
+      "SUM(DISTINCT SUBJECT_ID)",
+    Distractor3: "DISTINCT COUNT(SUBJECT_ID)",
```

Comment:
all of the answers should be complete queries with the SELECT keyword included

Reply:
just made the update to include the SELECT keyword